File: conformance-tests/flow-types/generic-type-annotation/generic-type-annotation-qualified-identifer-with-comments_m6.js
const foo: /* Before A */A/* After A */./* Before B */B/* After B */./* Before C */C/* After C */ = bar

function myFunc() : /* Before React */React/* After React */./* Before Node */Node/* After Node */./* Before Value */Value/* After Value */ { }

function myFunc2(baz : /* Before React */React/* After React */./* Before Node */Node/* After Node */./* Before Value */Value/* After Value */ ) : /* Before React */React/* After React */./* Before Node */Node/* After Node */./* Before Value */Value/* After Value */ { }

function myFunc3(baz : /* Before React */React/* After React */./* Before Node */Node/* After Node */./* Before Value */Value/* After Value */, biz : /* Before React */React/* After React */./* Before Node */Node/* After Node */./* Before Value */Value/* After Value */./* Before AnotherType */AnotherType/* After AnotherType */) : /* Before React */React/* After React */./* Before Node */Node/* After Node */./* Before Value */Value/* After Value */./* Before AnotherType */AnotherType/* After AnotherType */ { }

File: conformance-tests/flow-types/generic-type-annotation/generic-type-annotation-qualified-identifer-with-comments_m6.lua
local foo:  --[[ Before A ]]--[[ Before B ]]--[[ Before C ]]
A_B_C--[[ After A ]]--[[ After B ]]
--[[ After C ]]
 =
	bar
local function myFunc(
):  --[[ Before React ]]--[[ Before Node ]]--[[ Before Value ]]
React_Node_Value
	--[[ After React ]]
	--[[ After Node ]]
	--[[ After Value ]]
end
local function myFunc2(
	baz:  --[[ Before React ]] --[[ Before Node ]]	--[[ Before Value ]]
React_Node_Value
	--[[ After React ]]
	--[[ After Node ]]
	--[[ After Value ]]
):  --[[ Before React ]]--[[ Before Node ]]--[[ Before Value ]]
React_Node_Value
	--[[ After React ]]
	--[[ After Node ]]
	--[[ After Value ]]
end
local function myFunc3(
	baz:  --[[ Before React ]] --[[ Before Node ]]	--[[ Before Value ]]
React_Node_Value --[[ After React ]]	--[[ After Node ]]
	--[[ After Value ]]
,
	biz:  --[[ Before React ]] --[[ Before Node ]]	--[[ Before Value ]]
	--[[ Before AnotherType ]]
React_Node_Value_AnotherType
	--[[ After React ]]
	--[[ After Node ]]
	--[[ After Value ]]
	--[[ After AnotherType ]]
):  --[[ Before React ]]--[[ Before Node ]]--[[ Before Value ]]
--[[ Before AnotherType ]]
React_Node_Value_AnotherType
	--[[ After React ]]
	--[[ After Node ]]
	--[[ After Value ]]
	--[[ After AnotherType ]]
end
